server/src/controllers/surveyResultController.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import SurveyResultRequest from '../models/surveyResultRequest';
+import { SurveyResultRequestModel } from '../interfaces/surveyModel';
+import {
+  virtualVisitsDatabaseName,
+  surveyContainerName,
+  cosmosClient,
+  createDatabase,
+  createContainer
+} from '../utils/cosmosClient';
+
+export const storeSurveyResult = async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+  try {
+    const { body: requestData } = req;
```

Comment:
[Destructuring assignment - Assigning to new variable names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#:~:text=Copy%20to%20Clipboard-,Assigning%20to%20new%20variable%20names,-A%20property%20can)